src/components/smart-table-content/smart-table-content.md

```diff
@@ -0,0 +1,118 @@
+SmartTableComponent:
+
+      const React = require('react');
+      const pageSize = 3;
+
+      const columns = [{
+        baseKey: 'name',
+        headerLabel: 'Name',
+        useBase: true,
+        sortable: true,
+        type: 'string'
+      },{
+        key: 'species',
+        baseKey: 'description',
+        headerLabel: 'Species',
+        useBase: false,
+        sortable: false,
+        type: 'string'
+      },{
+        key: 'color',
+        baseKey: 'description',
+        headerLabel: 'Color',
+        useBase: false,
+        sortable: false,
+        type: 'fancyString',
+      }]
+      const rowData = [[{
+        id: 1,
+        name: 'Bob',
+        description: {
+          species: 'cat',
+          color: 'blue',
+        }
+      },{
+        id: 2,
+        name: 'Alice',
+        description: {
+          species: 'dog',
+          color: 'brown',
+        }
+      },{
+        id: 3,
+        name: 'Karen',
+        description: {
+          species: 'cat',
+          color: 'green',
+        }
+      }],[{
+        id: 4,
+        name: 'Jerome',
+        description: {
+          species: 'giraffe',
+          color: 'black',
+        }
+      },{
+        id: 5,
+        name: 'Steve',
+        description: {
+          species: 'horse',
+          color: 'pink',
+        }
+      }]]
+      const nrResults = 5;
+
+      const localization = {
+          noData: "No data for you!",
+          prevButton: "Previous",
+          nextButton: "Next",
+          caption: "Smart table",
+          expand: "Expand",
+          sorting: {
+            asc: "ascending",
+            desc: "descending",
+            ariaLabel: ({ field, dir }) =>
+            `sorting on ${field} in ${dir} order`
+          }
+         };
+
+
+        const getTdContent = (col, row) => {
```

Comment:
A function that always returns jsx is a a component and should be named as such I think.